src/db/plugins/upnp/ContentDirectoryService.cxx

```diff
@@ -77,8 +77,8 @@ ContentDirectoryService::readDirSlice(UpnpClient_Handle hdl,
 		{"BrowseFlag", "BrowseDirectChildren"},
 		{"Filter", "*"},
 		{"SortCriteria", ""},
-		{"StartingIndex", fmt::format_int{offset}.c_str(),
-		{"RequestedCount", fmt::format_int{count).c_str()},
+		{"StartingIndex", fmt::format_int(offset).c_str()},
```

Comment:
actually why is fmt even used here and not std::to_string?

Reply:
In this case, a `std::string` gets allocated anyway, so there's no advantage of using fmt's fixed-size stack buffer; but just always using fmt seems like a sane choice; std is inferior more often than not.

Reply:
I assume in the future MPD will migrate to std::format. Makes sense to reduce dependency on fmt.